### DISCUSSION TITLE
Is a phone required or is there a way to run this in a browser on my desktop computer?

### DISCUSSION THREAD

**Discussion by nathanturczan** [AR-js-org/AR.js#245] [Q&A] [answered]
I would like to use OBS to stream. Running this on my desktop with a webcam is more of an ideal setup, although I get it if the accelerometer data from the phone is required.

**Comment by jonnypage** [comment, marked as answer]
If you just want to show AR on a target, you can use a webcam. I'd suggest checking out the [documentation ](https://ar-js-org.github.io/AR.js-Docs/)and [examples ](https://github.com/AR-js-org/AR.js#-image-tracking)provided. They work fine on a computer's webcam.

A phone is required for geolocation, as it has the senors needed.

**Reply by nathanturczan** [reply]
Awesome, thanks!
